Library/Homebrew/debrew.rb

```diff
@@ -111,7 +111,7 @@ def self.debug(e)
 
     begin
       puts "#{e.backtrace.first}"
-      puts "#{Tty.red}#{e.class.name}#{Tty.reset}: #{e}"
+      puts Formatter.error("#{e.class.name}: #{e}")
```

Comment:
Please ignore my most recent comment; it doesn't make much sense after reviewing the code once more. https://github.com/Homebrew/brew/pull/823#r79279639 is a much saner suggestion.